cmd/generic-handlers.go

```diff
@@ -274,7 +274,7 @@ func setCorsHandler(h http.Handler) http.Handler {
 		AllowedOrigins:   []string{"*"},
 		AllowedMethods:   defaultAllowableHTTPMethods,
 		AllowedHeaders:   []string{"*"},
-		ExposedHeaders:   []string{"ETag"},
+		ExposedHeaders:   []string{"ETag", "Content-Length"},
```

Comment:
Can you add Date header as well? @krishnasrinivas - AWS indicates that all [Common Response Headers](http://docs.aws.amazon.com/AmazonS3/latest/API/RESTCommonResponseHeaders.html) can be exposed. 

We can perhaps Expose all headers here. 

> ### ExposeHeader Element
Each ExposeHeader element identifies a header in the response that you want customers to be able to access from their applications (for example, from a JavaScript XMLHttpRequest object). For a list of common Amazon S3 response headers, go to Common Response Headers in the Amazon Simple Storage Service API Reference guide.

Reply:
ping @krishnasrinivas did you see my above comment?